android/app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
         tools:node="remove"/>
     <uses-permission android:name="android.permission.CAMERA" android:required="false" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
```

Comment:
Why is this needed now (when apparently it wasn't previously)?

Reply:
Earlier it wasn't needed because React Native automatically called it for the first time,but once declined it has to be called manually.